xml/admin_saltcluster.xml

```diff
@@ -205,6 +205,764 @@ ses-min2   /dev/vdc  hdd  8192M  450575 True
    </note>
   </sect2>
 
+  <sect2 xml:id="drive-groups">
+   <title>Adding OSDs using &drvgrps; Specification</title>
+   <para>
+    <emphasis>&drvgrps;</emphasis> specify the layouts of OSDs in the &ceph;
+    cluster. They are defined in a single YAML file. In this section, we will
+    use <filename>drive_groups.yml</filename> as an example.
+   </para>
+   <para>
+    An administrator should manually specify a group of OSDs that are
+    interrelated (hybrid OSDs that are deployed on a mixture of HDDs and SDDs)
+    or share identical deployment options (for example, the same object store,
+    same encryption option, stand-alone OSDs). To avoid explicitly listing
+    devices, &drvgrps; use a list of filter items that correspond to a few
+    selected fields of <command>ceph-volume</command>'s inventory reports.
+    &cephadm; will provide code that translates these &drvgrps; into actual
+    device lists for inspection by the user.
+   </para>
+   <para>
+    To apply OSD specification to your cluster, run
+   </para>
+<screen>&prompt.cephuser;ceph orch apply osd -i <filename>drive_groups.yml</filename></screen>
+   <tip>
+    <para>
+     You can use the <option>--dry-run</option> option together with the
+     <command>ceph orch apply osd</command> command to see a preview of actions
+     that will actually happen when running the command without the
+     <option>--dry-run</option> option.
+    </para>
+   </tip>
+   <important>
+    <para>
+     All available clean disk devices that match the &drvgrps; specification
+     will be used as OSDs automatically after you add them to the cluster. If
+     you need to avoid this behavior, add the <option>--unmanaged=true</option>
+     option when creating OSDs:
+    </para>
+<screen>&prompt.cephuser;ceph orch apply osd -i <filename>drive_groups.yml</filename> --unmanaged=true</screen>
+    <para>
+     Alternatively, if you have already created OSDs without the
+     <option>--unmanaged=true</option>, you can disable the automatic OSD
+     creation using the following command:
+    </para>
+<screen>&prompt.cephuser;ceph orch osd spec --service-name  osd.all-available-devices --unmanaged</screen>
```

Comment:
The argument  to the `---service-name` parameter is not static. It needs to be in format of `service_type.service_id`

Reply:
does it apply to services other than `osd` as well?

Reply:
> The argument to the `---service-name` parameter is not static. It needs to be in format of `service_type.service_id`

Does it apply to services other than osd as well?